### DISCUSSION TITLE
Please add _passed hook to \Codeception\Module

### DISCUSSION THREAD

**Discussion by emulgeator** [Codeception/Codeception#6202] [Ideas]
#### What are you trying to achieve?
I tried to create a _passed hook method in one of my helper modules.
According to this changelog: https://codeception.com/02-27-2018/codeception-2.4-released.html
It should be possible but after some testing I realised that though `_failed` is in the Module class the `_passed` is not. 

### Details

* Codeception version: 2.5.6
* PHP Version: 7.2
* Operating System: Ubuntu
* Installation type: Composer

**Comment by Naktibalda** [comment]
That article says that `_passed` hook works in Cest classes.

**Reply by emulgeator** [reply]
Not sure I get what you mean. The `_failed` method is available in the `\Codeception\Module` class, and `_passed` is not

Can you show a simple example code where you hook on to a passed from a module?
